index.js

```diff
@@ -6,13 +6,13 @@ var JSONPersistenceProvider = require('./lib/jsonPersistenceProvider');
 var Utils = require('./lib/shared/utils');
 var ErrorInfo = require('./lib/error_info');
 var App = require('./lib/app.js');
-var RestAPI = require('./lib/restAPI');
+//var RestAPI = require('./lib/restAPI');
 module.exports = {
   Server,
   JSONPersistenceProvider,
   ServerPersistence,
   Utils,
   ErrorInfo,
   App,
-  RestAPI,
+  //RestAPI,
```

Comment:
I've added the file. For some reason (this was 3 years ago 😄 ) I had the file locally, and pushed it to npm, but forgot to add and push it here. So you can reenable this.

Reply:
added RestAPI again